### ISSUE TITLE
Long loading contacts due to continuos mtime updates

### ISSUE THREAD

**Issue by mitroko** [owncloud/contacts#763]
Hello, recently i've installed OC 7.0.4 and imported contacts from backup.
Contacts page loading too slow.
My 132 contacts now loading within 1-2 minutes (vresion OC 6.0.6 does it in 1 second).
I decided to debug and found that
php code tries to update every vcard in database (set mtime to datetime="current date + 35 days", in future)
And this slows down page load.

mysql sql log for updates populates with
97 Query UPDATE `oc_filecache` SET `mtime` = '1422798777', `etag`='54a068f9ea8de' WHERE `fileid` = '23'
97 Query UPDATE `oc_filecache` SET `mtime` = '1419796394', `storage_mtime`='1419796394' WHERE `fileid` = '23'
97 Query UPDATE `oc_filecache` SET `mtime` = '1419798778', `etag`='54a068fa5a6a2' WHERE `fileid` = '23'
97 Query UPDATE `oc_filecache` SET `mtime` = '1419796394', `storage_mtime`='1419796394' WHERE `fileid` = '23'
97 Query UPDATE `oc_filecache` SET `mtime` = '1422798778', `etag`='54a068fac4476' WHERE `fileid` = '23'
97 Query UPDATE `oc_filecache` SET `mtime` = '1419796394', `storage_mtime`='1419796394' WHERE `fileid` = '23'
97 Query     UPDATE `oc_filecache` SET `mtime` = '1419798779', `etag`='54a068fb361a7' WHERE `fileid` = '23'
97 Query UPDATE `oc_filecache` SET `mtime` = '1419796394', `storage_mtime`='1419796394' WHERE `fileid` = '23'
97 Query UPDATE `oc_filecache` SET `mtime` = '1422798779', `etag`='54a068fb8fdfa' WHERE `fileid` = '23'
and so on.
I'm the one user ov this OC installation and have all necessary backups to debug/drop/create.
Please help

**Comment by ogasser** [comment]
I seem to be havin a related problem, but I have a few more contacts. For me the contacts web app does not even successfully load but I get the following PHP error due to memory exhaustion:
```
PHP message: PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted
(tried to allocate 4104 bytes) in /var/www/owncloud/lib/private/vobject/stringproperty.php 
on line 61" while reading response header from upstream, client: ***, server: ***, 
request: "GET /index.php/apps/contacts/addressbook/local/1/contacts HTTP/1.1", 
upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "***", 
referrer: "https://***/index.php/apps/contacts/"
```

**Comment by jancborchardt** [comment]
@babelouest @jbtbnl @LEDfan can you check this out? :)

**Comment by mitroko** [comment]
Yay, looks like you fixed it in 8.0.2
Well done. Thank you!!!

**Comment by jancborchardt** [comment]
@mitroko good to hear! Thanks for reporting issues! :)
